Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "asset-classification-smart-contract"
-version = "2.0.1"
+version = "2.1.0"
```

Comment:
Sorry to tell you earlier that we should go to 2.1.0 instead of 3.0.0 - after watching that semantic versioning lunch and learn, I feel like we actually should go to 3.0.0 for this one.  It's a majorly-breaking set of changes.

Reply:
Haha, yeah probably makes sense... can't hold too tightly to version numbers... this from the creator of SemVer https://tom.preston-werner.com/2022/05/23/major-version-numbers-are-not-sacred.html 😆